docs/Database.md

```diff
@@ -28,7 +29,8 @@ and are also listed here:
 
 Attribute Name | Description
 ---|---
-`github_team_name` | `String`; The team's github name
+`gtid` | `String`; The team's Github ID
```

Comment:
Is `gtid` used in the official Github documentation? If not, I think I would rather have this as `github_team_id` or something similar - `gtid` doesn't seem descriptive enough to me.